### ISSUE TITLE
Fresh install of Pop_OS! 19.10, Docker fails to install

### ISSUE THREAD

**Issue by lpeabody** [docksal/docksal#1279]
Ran into an issue when installing Docksal on a fresh install of Pop_OS! 19.10. Docker fails to install.

```
lpeabody@pop-os:~$ bash <(curl -fsSL https://get.docksal.io)
[STEP 1/4] Updating fin...
fin 1.93.0 downloaded...
[STEP 2/4] Updating stack files...
overrides-ide.yml
services.yml
stack-acquia.yml
stack-default.yml
stack-default-nodb.yml
stack-node.yml
stack-pantheon.yml
volumes-bind.yml
volumes-nfs.yml
volumes-none.yml
volumes-unison.yml
Configuring network settings...
[STEP 3/4] Updating tools...
Installing Docker...
# Executing docker install script, commit: f45d7c11389849ff46a6b4d94e0dd1ffebca32c1
+ sudo -E sh -c apt-get update -qq >/dev/null
+ sudo -E sh -c DEBIAN_FRONTEND=noninteractive apt-get install -y -qq apt-transport-https ca-certificates curl >/dev/null
+ sudo -E sh -c curl -fsSL "https://download.docker.com/linux/ubuntu/gpg" | apt-key add -qq - >/dev/null
Warning: apt-key output should not be parsed (stdout is not a terminal)
+ sudo -E sh -c echo "deb [arch=amd64] https://download.docker.com/linux/ubuntu eoan stable" > /etc/apt/sources.list.d/docker.list
+ sudo -E sh -c apt-get update -qq >/dev/null
INFO: Searching repository for VERSION '19.03.4'
INFO: apt-cache madison 'docker-ce' | grep '19.03.4.*-0~ubuntu' | head -1 | awk '{$1=$1};1' | cut -d' ' -f 3

ERROR: '19.03.4' not found amongst apt-cache madison results

usermod: group 'docker' does not exist
sudo: docker: command not found
Docker installation/upgrade has failed.
```

**Comment by lpeabody** [comment]
```
lpeabody@pop-os:~$ apt-cache madison docker-ce
 docker-ce | 5:19.03.6~3-0~ubuntu-eoan | https://download.docker.com/linux/ubuntu eoan/stable amd64 Packages
```

Does `REQUIREMENTS_DOCKER` need a bump from 19.03.4 to 19.03.6?

**Comment by achekulaev** [comment]
Maybe. Needs check on Ubuntu

**Comment by lmakarov** [comment]
Should be resolved in [v1.14.0](https://github.com/docksal/docksal/releases/tag/v1.14.0) - using docker v19.03.9 for Debian/Ubuntu systems.
